charts/catalog/templates/apiregistration.yaml

```diff
@@ -0,0 +1,14 @@
+{{- if .Values.useAggregator }}
```

Comment:
done.  explicitly defaulted to false in values.yaml (since setting up an aggregator in 1.6 requires a bit of extra work).